src/charactersheet/viewmodels/common/wizard/index.js

```diff
@@ -98,6 +98,9 @@ export class WizardViewModel extends ViewModel {
 
         this.campaignName = ko.observable();
         this.playerName = ko.observable();
+        this.setting = ko.observable();
+        this.motif = ko.observable();
```

Comment:
Other options for `motif` could be `Overall Tone` or `Mood`.

Reply:
I think Motif is fine IMO.